source/containers.tex

```diff
@@ -11236,7 +11236,11 @@
 requirements\iref{random.access.iterators},
 and
 meets the requirements for
-constexpr iterators\iref{iterator.requirements.general}.
+constexpr iterators\iref{iterator.requirements.general},
+whose \tcode{value_type} is \tcode{value_type} and
+whose \tcode{reference} type is \tcode{reference}.
```

Comment:
It's not clear what entities `\tcode{value_type}` and `\tcode{reference}` refer to here. Presumably we mean member type aliases of `iterator_traits<iterator>`, so we could say `such that \tcode{iterator_traits<iterator>::value_type} is \tcode{value_type}` (and similarly for `reference`) but I think it would be clearer to simply use prose "value type" and "reference type" which name those properties of iterators:
```suggestion
whose value type is \tcode{value_type} and
whose reference type is \tcode{reference}.
```

Reply:
Looks good, but deviates from [string.view.iterators], which uses `value_type`. If the editors prefer this, I'd also like to extend the PR to [string.view.iterators] so that it also reads "value type".

Reply:
Yes, I think that would be an improvement. Having `\tcode{value_type}` mean different things two words apart is not good.